Vision/distance_regression.py

```diff
@@ -70,6 +72,9 @@ def calculate_angle(blob):
     relative_cx = blob.cx() - FRAME_HEIGHT
     relative_cy = blob.cy() - FRAME_WIDTH
     angle = math.degrees(math.atan(relative_cy/relative_cx))
+    if angle < 0:
+        angle += 180
+        return angle
     return angle
```

Comment:
The `return angle` inside the if is unnecessary as there is already another `return` below. Even more, it's up to you, but you can reduce these 4 lines two one with a ternary operator:
```python
return angle+180 if (angle < 0) else angle
```
Again, is your decision if you use a ternary operator or a normal `if` :)

Reply:
That looks good